### PR TITLE
Feature/support ssl policy in https load balancer

### PR DESCRIPTION
**Pull request by romankor**
Adding the ability to provide custom SSL policy to HTTP load balancer

### CODE REVIEW

**Comment by romankor** [comment]
@robmorgan ?

File: modules/http-load-balancer/variables.tf [hunk 1]
```diff
@@ -34,6 +34,12 @@ variable "ssl_certificates" {
   default     = []
 }
 
+variable "ssl_policy" {
+  description = "Link to ssl policy. Omitted if not provided"
+  type        = string
+  default     = ""
```
Comment (robmorgan) [inline modules/http-load-balancer/variables.tf:40]:
```suggestion
  default     = null
```
Let's set the default to `null` for better consistency with our other modules.

File: modules/http-load-balancer/variables.tf [hunk 2]
```diff
@@ -34,6 +34,12 @@ variable "ssl_certificates" {
   default     = []
 }
 
+variable "ssl_policy" {
+  description = "Link to ssl policy. Omitted if not provided"
```
Comment (robmorgan) [inline modules/http-load-balancer/variables.tf:38]:
```suggestion
  description = "A reference to the SslPolicy resource that will be associated with the TargetHttpsProxy resource. If not set, the TargetHttpsProxy resource will not have any SSL policy configured."
```
We'll adopt the TF provider docs as its clearer to the users.

**Review by robmorgan — CHANGES_REQUESTED** [review]
@romankor sorry for the delay! As a company, we've prioritized our AWS products during the pandemic this year and unfortunately, we have been **super slow** to respond to GCP issues. I've left some quick feedback on the PR. We should also add an explicit [test case](https://terratest.gruntwork.io) for this functionality to ensure it works correctly. Are you able to do this?

**Comment by romankor** [comment]
> @romankor sorry for the delay! As a company, we've prioritized our AWS products during the pandemic this year and unfortunately, we have been **super slow** to respond to GCP issues. I've left some quick feedback on the PR. We should also add an explicit [test case](https://terratest.gruntwork.io) for this functionality to ensure it works correctly. Are you able to do this?

@robmorgan 
fixed ,  any chances you guys will merge it now ?

**Comment by robmorgan** [comment]
Hi @romankor you will still need to add an explicit test case using [Terratest](https://terratest.gruntwork.io) so we can test this feature.. Are you able to do this?

**Comment by MoonMoon1919** [comment]
Closing this PR as it has been two years since last update and this repo has been in "sunsetting" for one year.

---

### Sunset notice
We believe there is an opportunity to create a truly outstanding developer experience for deploying to the cloud, however developing this vision requires that we temporarily limit our focus to just one cloud. Gruntwork has hundreds of customers currently using AWS, so we have temporarily suspended our maintenance efforts on this repo. Once we have implemented and validated our vision for the developer experience on the cloud, we look forward to picking this up. In the meantime, you are welcome to use this code in accordance with the open source license, however we will not be responding to GitHub Issues or Pull Requests.

If you wish to be the maintainer for this project, we are open to considering that. Please contact us at [support@gruntwork.io](mailto:support@gruntwork.io)

**Pull request closed without merging**